### ISSUE TITLE
Using Repeating Element throws Exception

### ISSUE THREAD

**Issue by zwenn** [Gernott/mask#400]
Hi,

when using the repeating element (mask 6.5.0 - 6.6.3 and Typo3 10.4.14 - 10.4.15) Typo3 throws attached exception.

In mask.json there are now two seperate parts for sql and tca as different entries:

"tx_mask_menu_siblings_pages": {
        "tca": {
            "tx_mask_menu_siblings_title": {
                "config": {
                    "autocomplete": "0",
                    "behaviour": {
                        "allowLanguageSynchronization": "0"
                    },
                    "default": "",
                    "eval": "",
                    "is_in": "",
                    "max": "",
                    "mode": "",
                    "placeholder": "",
                    "size": "",
                    "type": "input"
                },
                "description": "",
                "exclude": "1",
                "inlineParent": "tx_mask_menu_siblings_pages",
                "key": "menu_siblings_title",
                "l10n_mode": "",
                "label": "Abweichender Titel",
                "order": 2
            }
        }
    },
    "tx_mask_menu_siblings_tx_mask_menu_siblings_pages": {
        "sql": {
            "tx_mask_menu_siblings_title": {
                "tx_mask_menu_siblings_tx_mask_menu_siblings_pages": {
                    "tx_mask_menu_siblings_title": "tinytext"
                }
            }
        }
    },

After correcting this to:

"tx_mask_menu_siblings_pages": {
      "sql": {
            "tx_mask_menu_siblings_title": {
                "tx_mask_menu_siblings_pages": {
                    "tx_mask_menu_siblings_title": "tinytext"
                }
            }
        },
        "tca": {
            "tx_mask_menu_siblings_title": {
                "config": {
                    "autocomplete": "0",
                    "behaviour": {
                        "allowLanguageSynchronization": "0"
                    },
                    "default": "",
                    "eval": "",
                    "is_in": "",
                    "max": "",
                    "mode": "",
                    "placeholder": "",
                    "size": "",
                    "type": "input"
                },
                "description": "",
                "exclude": "1",
                "inlineParent": "tx_mask_menu_siblings_pages",
                "key": "menu_siblings_title",
                "l10n_mode": "",
                "label": "Abweichender Titel",
                "order": 2
            }
        }
    },

All worked fine again until the next saving.

Kind regards,
Sven

<img width="1143" alt="typo3-exception-be-2021-04-15" src="https://user-images.githubusercontent.com/6844841/114935075-02abba80-9e3b-11eb-86eb-f73f9fc21a16.png">

**Comment by nhovratov** [comment]
@zwenn Hi Sven, can you send me your mask.json so I can investigate?

And some more questions:
- Did you update from a older version of mask / TYPO3?
- Did you manipulate other parts manually in mask.json?
- Any other special things you did with mask?

I somehow need to reproduce the error, would be awesome if you can help.

**Comment by zwenn** [comment]
Hi Nikita,

thank you for your reply.

In the meantime I did some research on my own.
I made a clean install of Typo3 10.4.14 and mask 6.6.3 without any other extension.
Then I created the mask element: menu_subpages with the repeating element menu_subpages_items. Then the exception was thrown an the currupted mask.json was created.

Mask elements with other namings are working fine.

I attached a zip-file with correct_- and incorrect_mask.json

Thanx for your help.

[Archiv.zip](https://github.com/Gernott/mask/files/6329441/Archiv.zip)
